bin/spice/pkg/cli/cmd/login.go

```diff
@@ -150,6 +157,51 @@ spice login dremio --username <username> --password <password>
 	},
 }
 
+func dremioAuth(username string, password string) bool {
+
+	type userCredentials struct {
+		UserName string `json:"userName"`
+		Password string `json:"password"`
+	}
+
+	// TODO: Allow user to specify their own dremio host. Needs UX design for how the command should handle.
+	// TODO: Do we ultimately want to use dremio token instead of username/password in the following dremio requests?
+	const dremioHost = "dremio-4mimamg7rdeve.eastus.cloudapp.azure.com:9047"
+	dremioLoginAddr := fmt.Sprintf("http://%s/apiv2/login", dremioHost)
+
+	creds := userCredentials{
+		UserName: username,
+		Password: password,
+	}
+
+	jsonData, err := json.Marshal(creds)
+	if err != nil {
+		fmt.Println("Error marshaling JSON:", err)
+	}
+
+	req := fasthttp.AcquireRequest()
+	defer fasthttp.ReleaseRequest(req)
+
+	req.SetRequestURI(dremioLoginAddr)
+	req.Header.SetMethod("POST")
+	req.Header.SetContentType("application/json")
+	req.SetBody(jsonData)
+
+	resp := fasthttp.AcquireResponse()
+	defer fasthttp.ReleaseResponse(resp)
+
+	err = fasthttp.Do(req, resp)
+	if err != nil {
+		fmt.Println("Error sending request:", err)
+	}
+
+	if resp.StatusCode() == fasthttp.StatusOK {
+		return true
+	}
+
+	return false
```

Comment:
Don't return a bool. Align to Go conventions. Return an error if it fails and nil if it succeeds. or both a bool and error.